### DISCUSSION TITLE
Integrate jsonnet-builder into tanka

### DISCUSSION THREAD

**Discussion by dadav** [grafana/tanka#980] [Ideas]
Jsonnet-builder seems to be unmaintained (https://github.com/jsonnet-bundler/jsonnet-bundler/issues/186). As jb is kind of important to the tanka workflow, wouldn't it be great to integrate it into tanka? Like `tk tool jb`

What do you think?

**Comment by Duologic** [comment]
You are asking around holiday period, many of us involved are not here to respond. For what I can tell the project is still maintained, just doesn't need much maintenance.

Are there any specific issues that you think need of some attention?

(Edit: I meant to respond on the issue you linked)
